lib/toSyllabicSymbols.js

```diff
@@ -0,0 +1,144 @@
+const toUnicode = require( './toUnicode' )
+const { getRegexGroup } = require( './regex-utils' )
+
+// Some symbols represent multiple characters in a single unicode entity
+// These should be considered unsplittable and unmodifiable by other vowels/signs
+// Therefore can calculate their syllabic weight as is
+const oneUnitSyllables = [ 'ਇ', 'ਉ' ]
+const twoUnitSyllables = [ 'ਊ', 'ਓ', 'ਈ', 'ਏ', 'ਐ', 'ਆ', 'ਔ', 'ਆਂ', 'ਨੂੰ' ]
+
+// Other symbols/signs have no weight, therfore remove before processing
+const zeroUnitCharacters = [
+  '੍ਰ',
+  'ੵ',
+  'ੑ',
+  '੍ਵ',
+  '੍ਹ',
+  '੍ਟ',
+  '੍ਨ',
+  '੍ਯ',
+  '੍ਚ',
+  '੍ਤ',
+  '੍',
+  'ਿ',
+  'ੁ',
+  'ਂ',
+  'ਃ',
+  '।',
+  '॥',
+  '☬',
+  '਼',
+  '❁',
+]
+
+// For the rest, need to analyze the string
+// Any base character on it's own counts as one syllable
+// Adding any number of long/deeragh sounds will count as two syllables
+const syllableIdentifier = {
+  baseCharacter: 'b',
```

Comment:
Why not add the 1 and 2 here? It'll make more semantic sense to reference these values with a descriptive key later on, than floating 1s and 2s